spec/models/manageiq/providers/vmware/infra_manager/vm_spec.rb

```diff
@@ -49,7 +50,25 @@
     let(:op) { 'shutdown_guest' }
 
     context "when powered off" do
-      let(:vm_status) { {:raw_power_state => power_state_suspended} }
+      let(:vm_status) { {:raw_power_state => power_state_off} }
+
+      it 'is not available if tools is not installed' do
+        vm_status[:tools_status] = 'toolsNotInstalled'
+        vm.update(vm_status)
```

Comment:
This is an odd way to do this rather than `let()` and setting values in the `FactoryBot.create` call but wanted to stay consistent for now